templates/base/resources/usr/local/bin/attach_ubuntu_pro_subscription.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+if [ -n "${1}" ]
+then
+	LIFERAY_DOCKER_UBUNTU_PRO_TOKEN="${1}"
```

Comment:
@zsoltbalogh, where do you plan to set this info? Is it intended to our customers use this variable?

Reply:
Please check the ticket description. The customer would bring it.